### PR TITLE
feat: Integrate sqlalchemy, redis and celery in sentry

### LINKED ISSUE #6448: Add celery, redis and SQLAlchemy integrations in sentry
- [ ] Celery integration
- [ ] Redis Integration
- [ ] SQLAlchemy Integration

### PR DESCRIPTION
**Pull request by codedsun**
Fixes #6448 : integrated sqlalchemy,redis and celery in sentry

#### Changes proposed in this pull request:

- Integrated SQLAlchemmy, Redis, celery in sentry

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by codedsun** [comment]
@iamareebjamal Please check

**Comment by codedsun** [comment]
@iamareebjamal You need to guide me more on how do I activate celery and sqlAlchemy, could you give me the resources. For sentry integration, it just only says to setup this

```
sentry_sdk.init(app.config['SENTRY_DSN'], integrations=[SqlalchemyIntegration()])
sentry_sdk.init(app.config['SENTRY_DSN'], integrations=[CeleryIntegration()])
```

Please guide me further, on what needs to be done. Thanks

**Comment by iamareebjamal** [comment]
https://www.tutorialspoint.com/python/python_lists.htm

**Comment by codedsun** [comment]
@iamareebjamal damn, the shit I was doing.. ;(

File: app/__init__.py [hunk 1]
```diff
@@ -44,6 +44,13 @@
 from app.templates.flask_ext.jinja.filters import init_filters
 import sentry_sdk
 from sentry_sdk.integrations.flask import FlaskIntegration
+<<<<<<< HEAD
+from sentry_sdk.integrations.redis import RedisIntegration
```
[the remark this answers is not in the document]
Comment (iamareebjamal) [reply app/__init__.py]: Fix this, don't mark as resolved

File: app/__init__.py [hunk 2]
```diff
@@ -44,6 +44,13 @@
...
+=======
+from sentry_sdk.integrations.celery import CeleryIntegration
```
[the remark this answers is not in the document]
Comment (iamareebjamal) [reply app/__init__.py]: No need to write each import in separate line

File: app/__init__.py [hunk 3]
```diff
@@ -44,6 +44,13 @@
...
+from sentry_sdk.integrations.redis import RedisIntegration
+from sentry_sdk.integrations.sqlalchemy import SqlalchemyIntegration
```
[the remark this answers is not in the document]
Comment (iamareebjamal) [reply app/__init__.py]: Move module level imports to top

File: app/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (codedsun) [reply app/__init__.py]: @iamareebjamal It has been outdated

**Pull request merged**